dc/council/code/titles/28/sections/28~3152.xml

```diff
@@ -41,44 +41,46 @@
   <para>
     <num>(g)</num>
     <text>The written demand for payment required by subsection (f) of this section shall be in substantially the following form and shall be printed in at least 10-point type:</text>
-    <para>
-      <heading>“DEMAND FOR PAYMENT OF DISHONORED CHECK, DRAFT, ORDER, OR OTHER INSTRUMENT FOR THE PAYMENT OF MONEY</heading>
-    </para>
-    <para>
-      <heading>Notice: You may be sued 30 days after this notice if you don’t make payment within 30 days of this notice</heading>
-      <text>DATE:</text>
-      <text>TO:     Name of Drawer</text>
-      <text>Last known residence address</text>
-      <text>or place of business</text>
-      <text>YOUR CHECK, DRAFT, ORDER, OR OTHER INSTRUMENT FOR THE PAYMENT OF MONEY IN THE AMOUNT OF $ ____    , DATED ________    , PAYABLE TO THE ORDER OF ____________     HAS BEEN DISHONORED BY THE BANK/DEPOSITORY UPON WHICH IT WAS DRAWN, BECAUSE:</text>
-      <text>( )        YOU HAD NO ACCOUNT WITH THAT BANK/DEPOSITORY</text>
-      <text>( )        YOU HAD INSUFFICIENT FUNDS OR CREDIT WITH THAT BANK/DEPOSITORY</text>
-      <text>( )        A STOP PAYMENT ORDER WAS ISSUED</text>
-      <text>( )        OTHER (specify)</text>
-      <text>IF YOU DO NOT MAKE PAYMENT, YOU COULD BE SUED UNDER SECTION ____     OF THE DISTRICT OF COLUMBIA OFFICIAL CODE TO RECOVER PAYMENT. IF JUDGMENT IS RENDERED AGAINST YOU IN COURT, IT MAY ALSO INCLUDE NOT ONLY THE ORIGINAL FACE AMOUNT OF THE CHECK, BUT ALSO ADDITIONAL DAMAGES, AS FOLLOWS:</text>
-      <para>
-        <num>(1)</num>
-        <text>TWO TIMES THE FACE AMOUNT OF THE CHECK, OR $100, WHICHEVER IS GREATER;</text>
-      </para>
-      <para>
-        <num>(2)</num>
-        <text>COSTS; AND</text>
-      </para>
-      <para>
-        <num>(3)</num>
-        <text>REASONABLE ATTORNEY FEES.</text>
-      </para>
-    </para>
-    <para>
-      <heading>NAME OF PAYEE:</heading>
-    </para>
-    <para>
-      <heading>(PHONE NUMBER)</heading>
-    </para>
-    <para>
-      <heading>ADDRESS TO WHICH PAYMENT SHOULD BE DELIVERED</heading>
-      <text>IF YOU DISPUTE ANY OF THE FACTS LISTED ABOVE, CONTACT THE PAYEE IMMEDIATELY.”.</text>
-    </para>
+    <text>
+      <center>
+        <strong>“DEMAND FOR PAYMENT OF DISHONORED CHECK, DRAFT, ORDER, OR OTHER INSTRUMENT FOR THE PAYMENT OF MONEY</strong>
+        <br/>
```

Comment:
> Should the <br/> be removed and treated as a separate text tag?

Either is fine; leaving as-is.

>Also why does this section contain a ~ instead of a -?

The "~" (tilde) stands in for a ":" (colon) in the file name since Windows systems cannot have a ":" (colon) in the file name.